src/Stratis.Features.FederatedPeg/TargetChain/CrossChainTransferStore.cs

```diff
@@ -1083,7 +1083,8 @@ public ICrossChainTransfer[] QueryTransfersByStatus(CrossChainTransferStatus[] s
                 uint256[] partialTransferHashes = depositIds.ToArray();
```

Comment:
This method should call `GetTransfersByStatusInternalLocked` instead of duplicating the code here.

Reply:
I considered that, but it looks like that method is always calling `ValidateCrossChainTransfers`. I wasn't sure if we wanted to do that here because this is only for the UI.

Reply:
Nevermind I see there's an option to disable the validation so I'll just change it to do that.

Reply:
On a related note, we call `Synchronize()` and then `ValidateCCTs` way too much. `ValidateCCTs` happens already inside `Synchronize()`